internal/kafka/internal/cloudproviders/types.go

```diff
@@ -0,0 +1,72 @@
+package cloudproviders
+
+import "strings"
+
+// CloudProviderID represents
+type CloudProviderID string
```

Comment:
This PR also makes cloud provider IDs its own type. Instead of using strings for cloud provider ids in method parameters we should use this type.

Reply:
@bf2fc6cc711aee1a0c2a/control-plane take note